api/documentation/docs/tutorials/create-a-collective-offer.md

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 3
+---
+
+# Create a collective offer
+
+## Setup
+
+First, setup a provider, if not already done (more information [**here**](/docs/mandatory-steps/request-a-provider-account#how-to-get-a-provider-account).
+
+Next step is to ensure that link your API key to at least one venue. Please note that inside the integration environment, all venues are considered known by Adage. This will not be the case in real life, please ensure that your linked venues can be used for collective offers.
```

Comment:
```suggestion
Next step is to ensure that your API key is linked to at least one venue. Please note that inside the integration environment, all venues are considered known by Adage. This will not be the case in real life, please ensure that your linked venues can be used for collective offers.
```
`Please note that inside the integration environment, all venues are considered known by Adage. This will not be the case in real life, please ensure that your linked venues can be used for collective offers.` -> je mettrais cette partie dans une bulle warning ou info.